endorsement/shared_validation.py

```diff
@@ -54,6 +54,9 @@ def validate_shared_endorsers():
                 er.endorser = new_owner
                 er.accept_id = None
                 er.datetime_expired = None
+                er.datetime_notice_2_emailed = None
+                er.datetime_notice_3_emailed = None
+                er.datetime_notice_4_emailed = None
```

Comment:
Why do fields in a model need to be set to None? Is there a reason that the default value wouldn't be None/null?

Reply:
nulled since the 90 day expiration clock for the previous owner might have already started counting down

Reply:
ah, right.